new-backoffice/tutorials/creating-a-custom-dashboard/adding-functionality-to-the-dashboard.md

```diff
@@ -14,12 +14,12 @@ This is the third part of our guide to building a custom dashboard. This part co
 
 The steps we will go through in this part are:
 
-1. [Resources and services](adding-functionality-to-the-dashboard.md#1.-resources-and-services)
+1. [Contexts](adding-functionality-to-the-dashboard.md#1.-contexts)
 2. [Getting data from the server](adding-functionality-to-the-dashboard.md#2.-getting-data-from-the-server)
 
-## Step 1: Resources and services
+## Step 1: Contexts
 
-Umbraco has a large selection of resources and services that you can use in your custom property editors and dashboards. For this example, it would be nice to welcome the editor by name. To achieve this we can make use of the Umbraco resources.
+Umbraco has a large selection of contexts that you can use in your custom Property Editors and Dashboards. For this example, it would be nice to welcome the editor by name. To achieve this we can make use of the Umbraco Contexts.
```

Comment:
The example is about welcoming the editor by name so things should be kept simple and on point 💡 
```suggestion
Umbraco has a large selection of contexts that you can use in your custom Property Editors and Dashboards. For this example, we will welcome the editor by name. To achieve this we can make use of the Umbraco Contexts.
```